src/ima.js

```diff
@@ -876,9 +876,7 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
       adsRequest.linearAdSlotHeight = this.player.dimensions.height;
       adsRequest.nonLinearAdSlotWidth = this.player.dimensions.width;
       adsRequest.nonLinearAdSlotHeight = this.player.dimensions.height / 3;
-      if (this.getContentDuration() && !this.player.isLive()) {
-        adsRequest.contentDuration = this.getContentDuration();
-      }
+      adsRequest.contentDuration = -3;
```

Comment:
why -3?

Reply:
weirdos...
Anyway, set it to const so it will act as enum and be more understandable what's the meaning.
`const PRELOAD_NONE : number = -3;`